### PR TITLE
Fix Atom feed name

### PR DESCRIPTION
**Pull request by Psychpsyo**
This changes the advertised name of the Atom feed on every page from "Neos Modding Group" to "Neos Mod Releases".
It must replace the default _includes/head.html for that since the default is using ``{%- feed_meta -%}`` from the feed plugin (that we aren't using) which will always assume that the site title from _config.yml is also the feed title.
This replaces that with a manually constructed ``<link>`` element that has the actual feed's name on it so that feed readers will display it correctly.

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  {%- seo -%}
+  <link rel="stylesheet" href="{{ "/assets/css/style.css" | relative_url }}">
+  <link type="application/atom+xml" rel="alternate" href="https://neosmodloader.com/feed.xml" title="Neos Mod Releases">
+  {%- if jekyll.environment == 'production' and site.google_analytics -%}
+    {%- include google-analytics.html -%}
+  {%- endif -%}
```
Comment (ljoonal) [inline _includes/head.html:10]: ....IMO this could/should be yeeted xD

File: _includes/head.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  {%- seo -%}
+  <link rel="stylesheet" href="{{ "/assets/css/style.css" | relative_url }}">
+  <link type="application/atom+xml" rel="alternate" href="https://neosmodloader.com/feed.xml" title="Neos Mod Releases">
```
Comment (ljoonal) [inline _includes/head.html:7]: `{{ "/feed.xml" | relative_url }}`?

**Review by ljoonal — APPROVED** [review]

File: _includes/head.html (hunk 1 above)
Comment (Psychpsyo) [reply _includes/head.html:10]: I just took the template's default and only changed what I needed to.
But yea, I'd yeet it as well, lol

**Review by ljoonal — APPROVED** [review]
YOLO

**Pull request merged**